kubetest/azure.go

```diff
@@ -222,9 +222,11 @@ func (c *Cluster) populateApiModelTemplate() error {
 	}
 
 	// set default distro so we do not use prebuilt os image
-	v.Properties.MasterProfile.Distro = "ubuntu"
+	if v.Properties.MasterProfile.Distro == "" {
+		v.Properties.MasterProfile.Distro = "ubuntu"
+	}
 	for _, agentPool := range v.Properties.AgentPoolProfiles {
-		agentPool.Distro = "ubuntu"
+		agentPool.Distro = v.Properties.MasterProfile.Distro
```

Comment:
the distro may be different. It's better add the same check and only set it to `ubuntu` when it's empty